source/documentation/modules/meta.html.md.erb

```diff
@@ -97,6 +97,43 @@ title: sass:meta
 
 ## Functions
 
+<% function 'meta.calc-args($calc)', returns: 'list' do %>
+  <% impl_status dart: '1.40.0', libsass: false, ruby: false %>
+
+  Returns the arguments for the given [calculation].
+
+  [calculation]: ../values/calculations
+
+  If an argument is a number or a nested calculation, it's returned as that
+  type. Otherwise, it's returned as an unquoted string.
+
+  <% example(autogen_css: false) do %>
+    @debug meta.calc-args(calc(100px + 10%)); // unquote("100px + 10%")
```

Comment:
I agree that it's a bit unwieldy, but I think it's the best way we have of indicating that this returns an unquoted string with this particular content. Removing the `unquote()` would give the inaccurate impression that it returns a quoted string, and just writing `100px + 10%` seems like it could be ambiguous.

If we can come up with a better convention here I'm open to it, but let's do it after this lands—there are a number of existing functions that already use the `unquote()` convention, so we'd want to update them *en masse* anyway.

Reply:
Good point. `unquote` is definitely fine by me